mirage/src/main/resources/application.properties

```diff
@@ -15,6 +15,10 @@ management.metrics.export.influx.password=user
 management.metrics.export.influx.step=PT5s
 management.metrics.export.influx.uri=http://localhost:8086
 
+management.metrics.distribution.percentiles.all=.75,.90,.95,.99
```

Comment:
min, max, mean and p999 should also be added

Reply:
Added p999.
Maybe I misunderstand the request but min, max, mean is added on the grafana dashboard in the table view next to the chart. Do you need something in addition to that?

Reply:
This is fine.